### ISSUE TITLE
Missing packages.config (also outdated) and wrong NuGet packages path in the WinMergeContextMenu.vcxproj and .filters file

### ISSUE THREAD

**Issue by Andreas-Schniertshauer** [WinMerge/winmerge#985]
The file packages.config is missing in the project winmerge\ShellExtension\WinMergeContextMenu\WinMergeContextMenu.vcxproj
and
winmerge\ShellExtension\WinMergeContextMenu\WinMergeContextMenu.vcxproj.filters
file.
The Microsoft.Windows.ImplementationLibrary is outdated the new version is 1.0.210803.1

The packages path is wrong, the packages folder always resides parallel to the solution file so the entry in the vcxproj file should be ```<Import Project="..\..\packages ...``` instead of ```<Import Project="..\packages ...```
also the Error Condition.

**Comment by Andreas-Schniertshauer** [comment]
Ah, I see there exists an extra solution ShellExtension.vs2019.sln when opening this the path would be correct, but when opening ALL.vs2019.sln (as I did) the path is not correct.
In this case you need a nuget.config file that resides in the .sln solution folder, with the following content:
```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <config>
    <add key="repositoryPath" value=".\packages" />
  </config>
</configuration>
```
and
```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <config>
    <add key="repositoryPath" value="..\packages" />
  </config>
</configuration>
```
And in the WinMergeContextMenu.vcxproj the path must be ```..\..\packages```
than it works for both solutions.

**Comment by sdottaka** [comment]
Fixed with commit dfc9025 

Thank you for the report.

I changed microsoft/wil to be downloaded as a git submodule instead of as a NuGet package.
